4. User Guides/Editing Messages/README.md

```diff
@@ -53,3 +53,7 @@ Various emoji shortcuts exists to display popular text representations as graphi
 Graphical emojis can be enabled/disabled in [Your Account](../managing-your-account/) settings under the **Messages** section.
 
 When graphical emojis are disabled, emoji keywords and shortcuts will be displayed as is  e.g. `:smile:` or `:)`.
+
+## Sending audio recordings
+
+You can send audio recordings by clicking on the microphone icon on the right of the compose message box, which will start recording audio and replace the microphone icon with a square "stop" icon. To end the audio recording, click the "stop" icon, which will display a window where you can listen to and download the recording, name it and give it an optional description, before choosing to send or cancel and discard the audio recording.
```

Comment:
I think the *most frequent* users population of recorded audio clip will be on mobile (and Cordova mobile before our "+" series of native app become actually usable).   And in that workflow there are some variations.  The most annoying of which is , over say a 3g or 2.5g network, after recording there is an upload period that can be quite long - and status indicators that are often cryptic to decipher.